features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2059,7 +2059,10 @@
 </ResourceAccessControl>
 
     <ClientAppAuthentication>
-        <Application name="dashboard" hash="66cd9688a2ae068244ea01e70f0e230f5623b7fa4cdecb65070a09ec06452262"/>
+        <Application name="{{client_app.authentication.name}}" hash="{{client_app.authentication.hash}}" />
```

Comment:
using client_app.authentication prefix here would confuse with the client_apps.authentication. Can we use a known prefix here.
Something like, [dashboard_app.authentication] ?

Reply:
+1